website/versioned_docs/version-2.9.0/getting-started/docker.md

```diff
@@ -31,28 +31,80 @@ Additionally, if you wish to have the compiled email templates mounted, also rem
 
 ## How to use Docker
 
-**1. Build the containers using `docker-compose`**
+**1. Clone the [Saleor Platform](https://github.com/mirumee/saleor-platform) repository**
+
+```shell-session
+$ git clone git@github.com:mirumee/saleor-platform.git --recursive --jobs 3
+```
+
+The repo includes:
+* [Saleor Core (API)](https://github.com/mirumee/saleor)
+* [Saleor Dashboard](https://github.com/mirumee/saleor-dashboard)
+* [Saleor Storefront](https://github.com/mirumee/saleor-storefront)
+
+:::note
+To successfully clone the repo using SSH, make sure to add the SSH key to your ssh-agent.
+
+Start the SSH agent: `eval $(ssh-agent -s)`
+
+Add the key: `ssh-add /path/to/private/key`  
+
+Clone the repo: `git clone git@github.com:mirumee/saleor-platform.git --recursive --jobs 3`
+:::
```

Comment:
Would be nice if we could tackle this [issue](https://github.com/mirumee/saleor-platform/issues/19) all together.